### DISCUSSION TITLE
IOException at finished prestashop installation with cli

### DISCUSSION THREAD

**Discussion by Zudjo** [PrestaShop/PrestaShop#32470] [Q&A] [answered]
## Environment

- Windows 11 x64
- node 16.0.0
- npm 7.10.0
- xampp 8.1.17

## The exception

![image](https://user-images.githubusercontent.com/113335680/236671445-6719a4a0-38eb-4c02-a497-5d20f00a3d3f.png)

## Question

The installation is successful anyway, and the indicated directory IS empty.

How do I get rid of this exception?

**Comment by matks** [comment, marked as answer]
@Zudjo I think the problem is access control

The linux user you are using to un `php index_cli.php` must not be granted the rights to delete var/cache/dev

I suggest to put the linux user used for CLI and the linux user used for running the webserver in the same group
